PR_REVIEW_CHECKLIST.md

```diff
@@ -0,0 +1,98 @@
+      > 4. Verify that the bank account view opens without errors
+      >
+    - Take note of the distinction between testing _locally_ and _on staging_. Note: The staging site references the production version of the API.
+4. Make sure the testing steps also cover possible regressions.
+    1. Cases where we definitely need new regression tests:
+        1. Building a new page
+        2. Building a new feature
+        3. Building a new component?
+    2. Cases where we might need new regression tests:
+        1. Bug was easy to find
+5. Make sure that the test includes both **success** and **fail** scenarios.
+    - Ex: if the issue was to prevent the user from saving a field when it is empty, besides testing that we should also test that they can save the field _if it is not empty_.
+
+# Testing the changes
+
+1. Make sure there are no console errors related to the PR. If you find console errors in a branch you’re reviewing, check if they also exist in `main`.
+    1. If they do, proceed as normal (report them in #expensify-open-source if you think they need to be fixed).
+    2. If the errors do not exist in `main`, report it to the PR author - they need to be fixed before the PR can be merged.
+2. Test the changes on **all platforms**. Follow this guide (needs link) to set up each platform.
+    - If you’re unable to boot a platform for any reason, ask for help in the #expensify-open-source Slack channel. Your issue might already have been addressed there, so be sure to search first.
+3. Should we add regression testing for code introduced in this PR?
+    - If the PR introduces **new** pages, features, etc. then we **should** have new regression tests written. Please note this in your PR so that Applause can make sure it gets added.
+    - If the PR fixes a bug, Applause will check to make sure the existing regression testing steps already cover the workflow fixed by the bug.
+
+# Reviewing the code
+
+## Good code patterns to require
+
+1. Check that functions have comments when appropriate.
+    - If the function has params or returns something, use JSDocs syntax to describe them.
+        - Indicate the param name(s), data type passed and / or returned, and purpose if not immediately obvious.
+    - Obvious functions (with no params / return value) should not have comments.
+    - **In short: _Add comments & docs often, only when useful._**
+2. All copy / text shown to users in the product should be stored in the `src/languages/*` files for localization.
+3. Platform-specific files should follow the proper naming convention (see [Platform-Specific File Extensions](https://github.com/expensify/app#platform-specific-file-extensions))
+    - Example: `MyComponent/index.android.js`
+
+## Code patterns to check for improvement
+
+1. Is there appropriate error handling?
+2. Reuse...
+3. Are there any changes that could've been applied to one component that were applied to every usage of the component instead?
```

Comment:
I don't understand this instruction and it feels unnecessarily specific. Are we asking if there is some opportunity to remove redundant code specific to a React component? Why not just generally ask people to address opportunities to remove redundant code? This is basically just the DRY principle.

Reply:
> Are we asking if there is some opportunity to remove redundant code specific to a React component?

Yeah basically -> Checking if there's a way to refactor the component (or make a new one) to DRY up code around where the component is being used.

I think you're right it's basically DRY & a repeat of the "Considerations around code reuse" topic, so I think this can be removed.

Think it's useful to mention the DRY principle in this checklist or is it pretty self explanatory?

Reply:
Removing in favor of only including clear points